plugins/headless/hu.bme.mit.gamma.headless.api/src/hu/bme/mit/gamma/api/headless/Application.java

```diff
@@ -20,24 +20,47 @@ public Object start(final IApplicationContext context) throws Exception {
 		final Map<?, ?> args = context.getArguments();
 		final String[] appArgs = (String[]) args.get(IApplicationContext.APPLICATION_ARGS);
 
+		Level level = Level.INFO;
+
 		if (appArgs.length == 0) {
-			logger.log(Level.WARNING, "Arguments must be given! Either a \"workspace\", \"import\" or \"gamma\" argument is expected.");
+			logger.log(Level.WARNING,
+					"Arguments must be given! Either a \"workspace\", \"import\" or \"gamma\" argument is expected.");
 			return null;
 		} else {
+
+			switch (appArgs[1]) {
+			case "info":
+				level = Level.INFO;
+				break;
+			case "warning":
+				level = Level.WARNING;
+				break;
+			case "severe":
+				level = Level.SEVERE;
+				break;
+			case "off":
+				level = Level.OFF;
+				break;
+			default:
+				level = Level.INFO;
```

Comment:
This is the same as the `case "info":` Either delete that branch or throw an exception here.